hnn_core/params.py

```diff
@@ -657,27 +657,50 @@ def compare_dictionaries(d1, d2):
     return d1
 
 
-def convert_to_hdf5(params_fname, out_fname, include_drives=True,
-                    overwrite=True, write_output=False):
-    """Converts json or param format to hdf5
+def convert_to_json(params_fname,
+                    out_fname,
+                    network_connectivity='jones_2009_model',
+                    include_drives=True,
+                    overwrite=True):
+    """Converts legacy json or param format to hierarchical json format
```

Comment:
sorry I missed the discussion. Do you guys have a specification of the hierarchical json format? how will you store connectivity etc? Or is it an intermediate format before going to hdf5?

Reply:
The format is Network object converted to a dictionary format. So the connectivities are just the same hierarchical dictionary format as the Network Object. Pretty much everything is preserved except numpy arrays need to be converted to lists when written to json and converted back upon read-in. 

This is not an intermediate format before going to hdf5. All the network connections, drives, cell parameters can be saved and loaded with this format. 

What is not included with this format is the ability to save outputs (the optional recordings during simulation saved to the Network). [Is there a better term to use than "outputs"?] We plan to use the hdf5 format for the saving of network outputs. For that hdf5 file, a version of this json will be saved as a single dataset in the hdf5 (A string json binarized), that way the network configurations are paired with the outputs.